package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sgmap.ludwig",
+  "name": "sgmap-ludwig",
```

Comment:
A justification for the rename would be more than welcome. This impacts all reusers (even if there are none today, a note in the commit message seems mandatory to me).

If we rename, let's use [scoped packages](https://docs.npmjs.com/misc/scope) and publish this as `@sgmap/ludwig`.